src/ensembl/src/content/app/browser/BrowserCog.tsx

```diff
@@ -65,7 +70,9 @@ const BrowserCog: FunctionComponent<BrowserCogProps> = (
         return (
           item && (
             <animated.div key={key} style={style}>
-              <BrowserTrackConfig />
+              <div ref={ref}>
+                <BrowserTrackConfig />
+              </div>
```

Comment:
If the only purpose of this div is to provide an anchor point for a ref, then perhaps, instead of adding a div, we could use forwardRef on BrowserTrackConfig?

Reply:
I tried using `forwardRef` but I'm not able to make TypeScrips happy for some reason.
Although, this `div` here doesn't have much purpose, I feel if makes the code simpler than using `forwardRefs`. Do we really need `forwardRef` here?

Reply:
> Do we really need forwardRef here?

Maybe not :smiley: 
Out of curiosity, why do we add this hook and this ref in the BrowserCog component rather than in the BrowserTrackConfig component?

Reply:
By this way I'm just avoiding passing down `updateSelectedCog` to BrowserTrackConfig.

Reply:
That’s right. But doesn’t the ability to close when someone clicks outside a component more naturally belong to this component rather than to its parent?
In other words, isn’t it reasonable for  <BrowserTrackConfig /> component to receive the onClose (or handleClose, etc.) prop that it uses when it decides that it needs to close?

Reply:
That's a fair point. I've update it.